modules/claims_api/spec/workers/special_issue_updater_spec.rb

```diff
@@ -9,14 +9,18 @@
     Sidekiq::Worker.clear_all
   end
 
-  let(:user) { FactoryBot.create(:evss_user, :loa3) }
+  let(:user) do
+    {
+      'ssn' => '796043735'
+    }
+  end
```

Comment:
might be worth keeping the user factory generated here and then just build the hash to pass in off of that factory record. that way this ssn lives only in the factory, in case it ever needs to change or something.

Reply:
Good idea.
I had made a similar change over in the FlashUpdater spec the other day.  Went ahead and made this change there too.
[See](https://github.com/department-of-veterans-affairs/vets-api/pull/6171/commits/c0e9c1b08abbb50b770b9722c05760d89e9dba5b)